control_plane/src/storage_controller.rs

```diff
@@ -628,6 +628,10 @@ impl StorageController {
             args.push("--timelines-onto-safekeepers".to_string());
         }
 
+        if let Some(sk_cnt) = self.config.timeline_safekeeper_count {
```

Comment:
Hmm thinking more about this. We should probably, if the `timeline_safekeeper_count` param has not been specified, and the number of safekeepers configured is less than three, specify it to the number of safekeepers configured.

Reply:
Yes, that's the plan. But we need to wait till this option reaches the release branch to not break compatibility tests

So I'm waiting for Friday to fix it